pkg/config/cos.go

```diff
@@ -284,8 +283,6 @@ func SaveOriginalNetworkConfig() error {
 // - manipulates nameservers in `/etc/resolv.conf`.
 // - call `wicked ifreload all` if `run` flag is true.
 func UpdateNetworkConfig(stage *yipSchema.Stage, networks map[string]Network, run bool) error {
-	var staticDNSServers []string
```

Comment:
Just curious, do we intend to remove the ability to configure DNS server for LIVE OS?

Reply:
We need to make the DNS servers text field out of networks and save them to `OS.DNSNameservers`.

Reply:
@bk201 do we need to ask users to input DNS servers if they choose DHCP mode?

Reply:
Those are entered as static DNS servers, it also works with DHCP mode.

Reply:
@bk201 , we still can let users configure DNS servers in static mode. After confirming with @weihanglo , I think what he means is that "whether we let users have ability to connect to internet/intranet after they configure network? Instead of waiting util Install Panel". WDYT?

Reply:
We need DNS settings to be applied after the network pannel. NTP panel, ssh keys panel, and config panels need this.
Do you think moving the `DNS servers` Input widget out of static IP settings works?
![Screen Shot 2021-09-30 at 10 45 10 AM](https://user-images.githubusercontent.com/1691518/135377480-dfcf6982-692d-4a1d-acc1-a14b7b498d82.png)
l?

Reply:
Yes, that would make sense for some users and make it an optional config

Reply:
Okay. I will add an optional DNS panel if users choose static mode.

Reply:
> Okay. I will add an optional DNS panel if users choose static mode.

The optional DNS servers work for both DHCP and static IP modes.

Reply:
Added DNS optional panel. @bk201 @guangbochen @weihanglo please help me review again. Thank you.